pkg/ccl/kvccl/kvtenantccl/tenant_upgrade_test.go

```diff
@@ -298,13 +298,13 @@ func TestTenantUpgradeFailure(t *testing.T) {
 		initialTenantRunner := sqlutils.MakeSQLRunner(tenant)
 		// Ensure that the tenant works.
 		initialTenantRunner.CheckQueryResults(t, "SHOW CLUSTER SETTING version",
-			[][]string{{clusterversion.TestingBinaryMinSupportedVersion.String()}})
+			[][]string{{v0.String()}})
 		initialTenantRunner.Exec(t, "CREATE TABLE t (i INT PRIMARY KEY)")
 		initialTenantRunner.Exec(t, "INSERT INTO t VALUES (1), (2)")
 		// Upgrade the host cluster and have it crash on v1.
 		sqlutils.MakeSQLRunner(tc.ServerConn(0)).Exec(t,
 			"SET CLUSTER SETTING version = $1",
-			v2.String())
+			v1.String())
```

Comment:
This was the main change here unblocking the test, right? Looks like the commentary above also expected v1, so wonder when we regressed.